apps/mobile/TheApp/DB/Records.cs

```diff
@@ -26,20 +26,20 @@ public Records()
 
 		void RecordTx(Events.TxStatus status)
 		{
-			if (status.SignedResponse != null)
+			if (status.Response != null)
 			{
-				var txResponse = status.SignedResponse;
-				var (address, response) = TheUtils.Signing.GetSignerAndMessage<TapCapBrokerPurchase>(txResponse);
-				var (clientAddress, request) = TheUtils.Signing.GetSignerAndMessage<TapCapClientRequest>(response.SignedRequest);
+				//var txResponse = status.SignedResponse;
```

Comment:
The commented-out code is non-functional, but still required.  Basically, anything that wasn't 100% required was just stripped rather than upgraded.